src/fsw/FCCode/OrbitController.cpp

```diff
@@ -1,8 +1,20 @@
 #include "OrbitController.hpp"
+#include <fsw/FCCode/Estimators/rel_orbit_state_t.enum>
+#include <fsw/FCCode/Estimators/RelativeOrbitEstimator.hpp>
 
 const constexpr double OrbitController::valve_time_lin_reg_slope;
 const constexpr double OrbitController::valve_time_lin_reg_intercept;
 
+// Firing nodes
+constexpr double pi = gnc::constant::pi;
+static constexpr std::array<double, 3> firing_nodes_far = {pi/3, pi, -pi/3};
+static constexpr std::array<double, 18> firing_nodes_near = {pi/18, pi/6, pi*(5/18), pi*(7/18), pi/2, pi*(11/18), 
+                                pi*(13/18), pi*(5/6), pi*(17/18), -pi*(17/18),
+                                -pi*(5/6), -pi*(13/18), -pi*(11/18), -pi/2,
+                                -pi*(7/18), -pi*(5/18), -pi/6, -pi/18};
+
+auto gain_factor = (double)(firing_nodes_near.size() / firing_nodes_far.size());
```

Comment:
We actually need to cast first otherwise it's still going to do integer math and only then cast to a double. No reason not to make this `static constexpr` either.
```suggestion
static constexpr auto gain_factor = static_cast<double>(firing_nodes_near.size()) / firing_nodes_far.size();
```